man/systemd.network.xml

```diff
@@ -1676,6 +1676,23 @@
             then all options specified earlier are cleared. Defaults to unset.</para>
           </listitem>
         </varlistentry>
+
+        <varlistentry>
+          <term><varname>EnableSourceRoutingRules=</varname></term>
+          <listitem>
+            <para>This option is used to configure source routing for an interface.
+            When combined with setting an explicit route table (with the <literal>RouteTable=</literal> option),
+            this causes routing policy rules to be automatically be created for the DHCP obtained address on the interface,
+            of the form "from $address lookup $routetable".</para>
```

Comment:
hmm, this explains one low level language (our .network files), with another one (the iproute2 command line). It's only understandable if you know the letter well, and want to understand the former. I doubt this is necessarily a given though. Hence, can we express this in prose instead, please?

Reply:
should say "takes a boolean". also should say what the default is.